web/src/settings_users.js

```diff
@@ -668,12 +668,12 @@ export function show_edit_user_info_modal(user_id, $container) {
             .hide();
     }
 
-    const element = "#edit-user-form .custom-profile-field-form";
-    $(element).empty();
-    settings_account.append_custom_profile_fields(element, user_id);
-    settings_account.initialize_custom_date_type_fields(element);
+    const custom_profile_field_form = "#edit-user-form .custom-profile-field-form";
```

Comment:
I would put `selector` in this name, to make clear that this is a string format selector, not some form of object.

Reply:
``` diff
diff --git a/web/src/settings_users.js b/web/src/settings_users.js
index 5a7b795b8f..beab7abb09 100644
--- a/web/src/settings_users.js
+++ b/web/src/settings_users.js
@@ -668,12 +668,12 @@ export function show_edit_user_info_modal(user_id, $container) {
             .hide();
     }
 
-    const custom_profile_field_form = "#edit-user-form .custom-profile-field-form";
-    $(custom_profile_field_form).empty();
-    settings_account.append_custom_profile_fields(custom_profile_field_form, user_id);
-    settings_account.initialize_custom_date_type_fields(custom_profile_field_form);
+    const custom_profile_field_form_selector = "#edit-user-form .custom-profile-field-form";
+    $(custom_profile_field_form_selector).empty();
+    settings_account.append_custom_profile_fields(custom_profile_field_form_selector, user_id);
+    settings_account.initialize_custom_date_type_fields(custom_profile_field_form_selector);
     const fields_user_pills = settings_account.initialize_custom_user_type_fields(
-        custom_profile_field_form,
+        custom_profile_field_form_selector,
         user_id,
         true,
         false,
```